src/util/Numerical.js

```diff
@@ -234,10 +234,8 @@ var Numerical = new function() {
                     // We multiply with a factor to normalize the coefficients.
                     // The factor is just the nearest exponent of 10, big enough
                     // to raise all the coefficients to nearly [-1, +1] range.
-                    var mult = pow(10,
-                            abs(Math.floor(Math.log(gmC) * Math.LOG10E)));
-                    if (!isFinite(mult))
-                        mult = 0;
+                    var mult = gmC === 0 ? 0 : pow(10,
+                        abs(Math.floor(Math.log(gmC) * Math.LOG10E)));
```

Comment:
I tiny detail: We've been using double-indents to distinguish a broken statement from a nested code block (see left). This used to be a Java convnetion that found its way into paper.js and is in use across the code-base...